src/lib/github/addIssuesToRepo.ts

```diff
@@ -11,21 +12,35 @@ export default async function addIssuesToRepo (context: Context, newIssues: Issu
   const octokit = context.github;
   const { owner, repo } = getBasicRepoProps (context);
   const ghIssues = await getAllRepoIssues(context);
+
   newIssues.forEach(async ({title, body, authors}) => {
     const assignees = uniq(authors);
-    const currentGHIssue = find(ghIssue => ghIssue.title === title, ghIssues);
+
+    /**
+     * Am up to writing a function that removes checked out or no longer existing issue comments.
+     */
+
+    const currentGHIssueComments = find(ghIssueComment => ghIssueComment.comments === body, ghIssues);
+    const newGHIssueComments = currentGHIssueComments.comments.includes(" - [x] ") || !(currentGHIssueComments.comments === body) ? replaceCommentArray(currentGHIssueComments.comments, body) : currentGHIssueComments.comments;
```

Comment:
What you need to modify is the `body` value within the fields object. That's what gets used in editing or creation of an issue.